lib/templates.js

```diff
@@ -1,81 +1,112 @@
 'use strict';
 
-var api = 'templates'
-  , Promise = require('./Promise')
-  , toApiFormat = require('./toApiFormat');
+const api = 'templates';
+const Promise = require('./Promise');
+const _ = require('lodash');
 
 module.exports = function(client) {
-  var templates = {
-    all: function(callback) {
+  return {
+    /**
+     * List an overview of all templates.
+     *
+     * @param {RequestCb} [callback]
+     * @returns {Promise}
+     */
+    list: function(callback) {
       var options = {
         uri: api
       };
       return client.get(options).asCallback(callback);
     },
-    find: function(options, callback) {
+    /**
+     * Get details about a specified template by its id.
+     *
+     * @param {string} id
```

Comment:
string or String? I'll check jsdoc when online but in express source, I see it's String. 
```
/**
 * Check if the given `lang`s are acceptable,
 * otherwise you should respond with 406 "Not Acceptable".
 *
 * @param {String} ...lang
 * @return {String|Array}
 * @public
 */

req.acceptsLanguages = function(){
  var accept = accepts(this);
  return accept.languages.apply(accept, arguments);
};
```